pages/windows/bleachbit_console.md

```diff
@@ -0,0 +1,25 @@
+# bleachbit_console
+
+> The Bleachbit command-line interface (CLI) allows users to automate the process of cleaning junk files on their system.
+> It can be run with various command-line arguments to perform different clean-up operations.
+> More information: <https://docs.bleachbit.org/doc/command-line-interface.html>.
+
+- Perform the clean-up operation and delete files:
+
+`bleachbit_console.exe --clean`
```

Comment:
```suggestion
- Perform the clean-up operation and delete files:

`bleachbit_console --clean`
```

Optional fix: `.exe` can be removed.

Reply:
Generally, for Bleachbit in Windows `.exe` extension is required, and it's their recommended method so I will mark these suggestions as resolved. For using without exe you need to add it is an environment variable.